### ISSUE TITLE
HyperParameters of cifar

### ISSUE THREAD

**Issue by Holidays1999** [SHI-Labs/Compact-Transformers#37]
I want to reproduce your paper, however I find the weight decay and learning rate of cifar10 in *.yaml is different from paper's said, could your please tell me which parameter should I use?

**Comment by alihassanijr** [comment]
Hello,
Thank you for your interest.
We mentioned the default settings for all datasets in the paper. Every experiment has a different set of settings, and obviously not all of them would fit in the paper. Because of that, we released the yaml files, so that the exact training settings are available for those who would need to reproduce the results. Therefore, use the yaml files for any given model and dataset.

Feel free to re-open the issue if you have other questions.

**Comment by Holidays1999** [comment]
Thanks very much.
I will followed the settings in the paper.

**Comment by Holidays1999** [comment]
hello, 
I am sorry to brother you again. If I want to reproduce cct results in cifar10, could you please tell me I should use train.py or examples/main.py?
thank you very mush.

**Comment by alihassanijr** [comment]
Hi,
It depends on the results. If what you want to reproduce comes with a config file in `configs/pretrained`, use that configuration file with `train.py`. The rest are older results and can be reproduced with `examples/main.py`.

We are going to update the repository again soon with checkpoints and config files for all of our main results.

**Comment by Holidays1999** [comment]
Thanks!
And I also wonder whether you use ddp for cifar10 or not, when I follow your configs(300epochs) to run cifar10 on a single card, I find it something may be hard to converge.
